src/Elastic.Apm/Report/PayloadSenderV2.cs

```diff
@@ -123,7 +127,9 @@ public PayloadSenderV2(
 			_eventQueue = new BatchBlock<object>(configuration.MaxBatchEventCount);
 
 			SetUpFilters(TransactionFilters, SpanFilters, ErrorFilters, apmServerInfo, logger);
-			StartWorkLoop();
+
+			if (isEnabled)
```

Comment:
nit: for consistency, should we prefer the `_isEnabled` field rather than the parameter here? We use the parameter above.